app/controllers/performance_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PerformanceController < ApplicationController
+  def index
+    @comments = Comment.all
```

Comment:
This is going to be huge! You should be getting all the values you need to use in your view here, not just getting all the comments then still calling methods against them all in the view.